.buildkite/pipeline.yml

```
@@ -155,6 +155,19 @@ steps:
         upload:
           - maze_output/**/*
 
+  - label: "Node.js Integration Tests"
+    depends_on: build
+    commands:
+      - chmod +x bin/arm64-macos-bugsnag-cli
+      - bundle install
+      - bundle exec maze-runner --port=$((MAZE_RUNNER_PORT)) features/node
+    plugins:
+      artifacts#v1.5.0:
+        download:
+          - bin/arm64-macos-bugsnag-cli
+        upload:
+          - maze_output/**/*
```

Not a big deal for just 2 scenarios, but this could pay off longer term or in other situations that just copy/paste this step as an example:
```suggestion
          - maze_output/maze_output.zip
          - maze_output/failed/*
```